docs/simple_switch.md

```diff
@@ -53,21 +140,54 @@ not access directly.
 - `mcast_grp`: needed for the multicast feature. This field needs to be written
 in the ingress pipeline when you wish the packet to be multicast. A value of 0
 means no multicast. This value must be one of a valid multicast group configured
-through bmv2 runtime interfaces.
+through bmv2 runtime interfaces. See the "after-ingress pseudocode" for
+relative priority of this vs. other possible packet operations at
+end of ingress.
 - `egress_rid`: needed for the multicast feature. This field is only valid in
 the egress pipeline and can only be read from. It is used to uniquely identify
 multicast copies of the same ingress packet.
-- `resubmit_flag`: should not be accessed directly. It is set by the `resubmit`
-action primitive and is required for the resubmit feature. As a remainder,
-`resubmit` needs to be called in the ingress pipeline.
+- `resubmit_flag`: should not be accessed directly. It is set by the
+`resubmit` action primitive and is required for the resubmit
+feature. As a reminder, `resubmit` needs to be called in the ingress
+pipeline. See the "after-ingress pseudocode" for relative priority of
+this vs. other possible packet operations at end of ingress.
 - `recirculate_flag`: should not be accessed directly. It is set by the
 `recirculate` action primitive and is required for the recirculate feature. As a
-remainder, `recirculate` needs to be called from the egress pipeline.
+reminder, `recirculate` needs to be called from the egress pipeline.
+See the "after-egress pseudocode" for the relative priority of this
+vs. other possible packet operations at the end of egress processing.
+- `clone_spec`: should not be accessed directly. It is set by the
```

Comment:
Needs to be moved to `standard_metadata`

Reply:
I moved the description of clone_spec to the earlier list of standard_metadata fields.